build/charts/nephe/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+name: nephe
+type: application
+displayName: Nephe
+home: https://antrea.io/
+version: 0.0.0
+appVersion: latest
+kubeVersion: ">= 1.16.0-0"
+icon: https://raw.githubusercontent.com/antrea-io/antrea/main/docs/assets/logo/antrea_logo.svg
+description: Antrea managed security policies in the public cloud
+dependencies:
+- name: crds
+  condition: crds.enabled
+  version: 0.0.0
```

Comment:
IMO, since you do not have Custom Resources in your chart, having the CRDs in a dedicated chart is not necessarily the best approach. Is there a reason why you went that way / an existing project you used as a reference?

I am pointing this out because we discussed this at length for Antrea (https://github.com/antrea-io/antrea/issues/3665) and decided take a different approach, and there is value in keeping things consistent.

Reply:
On a related note, it is not clear to me how - in your specific case - using a dependency chart with all the CRDs is different from treating the CRDs as regular templated resources.

Reply:
I used metallb project as reference.. and helm documentation https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#method-2-separate-charts

Reply:
discussed offline.. sticking to sub-chart approach.